### PR TITLE
19109 Added caution to review step

### PR DESCRIPTION
**Pull request by severinbeauvais**
*Issue #:* bcgov/entity#19109

*Description of changes:*
- app version = 5.6.22
- updated base class to remove border radius (per UI designs)
- added message box to step 5

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the bcrs-entities-create-ui license (Apache 2.0).

Sample:

image

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-create-dev--pr-607-26klo4ll.web.app

File: src/views/AmalgamationRegular/ReviewConfirm.vue [hunk 1]
```diff
@@ -361,4 +370,9 @@ h2::before {
   counter-increment: header-counter;
   content: counter(header-counter) '. ';
 }
+
+// override base class text size
+.message-box p {
+  font-size: $px-16;
+}
```
Comment (severinbeauvais) [inline src/views/AmalgamationRegular/ReviewConfirm.vue:377]: I'm waiting to hear back from Yui on whether I should make this change to the base class (which affects the other message boxes) or just here.

File: src/assets/styles/base.scss [hunk 2]
```diff
@@ -272,7 +272,7 @@ p {
 .message-box {
   background-color: $BCgovGold0 !important;
   border-color: $BCgovGold5 !important;
-  border-radius: 0;
+  border-radius: 0 !important;
```
Comment (severinbeauvais) [inline src/assets/styles/base.scss:275]: This was being overridden by a Vuetify base class. Now it works.

File: src/views/AmalgamationRegular/ReviewConfirm.vue (hunk 1 above)
Comment (JazzarKarim) [reply src/views/AmalgamationRegular/ReviewConfirm.vue:377]: I think Yui won't be back until next week Sev.

**Review by JazzarKarim — APPROVED** [review]
👍

File: src/views/AmalgamationRegular/ReviewConfirm.vue (hunk 1 above)
Comment (severinbeauvais) [reply src/views/AmalgamationRegular/ReviewConfirm.vue:377]: Oh. In that case, let's leave it like this for now.

**Pull request merged**